packages/teams/src/utils/get-teams-available-in-product.ts

```diff
@@ -1,22 +1,20 @@
 import { HubProduct, IGroupTemplate } from "../types";
 import { cloneObject } from "@esri/hub-common";
 import { WELLKNOWNTEAMS } from "../well-known-teams";
-import { PRE_9_1_WELLKNOWNTEAMS } from "../pre-9-1-well-known-teams";
 
+// TODO: at next breaking change remove portalApiVersion param
 /**
  * Given a product, return the groups templates that are available
  * @param {string} product Product name portal | basic | premium
  */
 export function getTeamsAvailableInProduct(
   product: HubProduct,
-  portalApiVersion: string
+  portalApiVersion: string // DEPRECATED
 ) {
-  // TODO: remove this when needed (after may 1st 2021)
-  // choosing the type of well known team based on the current portal version
-  const teams =
-    parseFloat(portalApiVersion) < 9.1
-      ? PRE_9_1_WELLKNOWNTEAMS
-      : WELLKNOWNTEAMS;
+  /* tslint:disable no-console */
+  console.warn("portalApiVersion is deprecated and will be removed at v9.0.0");
```

Comment:
Add the function name to the warning... otherwise, it's hard to know where this is coming from